wiki/Tokens.md

```diff
@@ -1,9 +1,26 @@
+### About
 Tokens are shapes that can be added to the map representing things such as creatures, players, items, area effects, notes or anything else you can think of.
+### Usage
+To add a token, click on the token tool on the left control panel and then click anywhere on the map to place the token. The token size will default to the grid size if it is present. <br> To customize a token,right click on a token to reveal the token menu. From the token menu you can name the token, change its size and color, hide/show the token to players, and link a note.
+### Description of Features on the Token Menu
+` Label `  this is the token identifier.It can be set to any value either a string or a number with the default value being 1.<br>
 
-Click on the token tool on the left and then click somewhere on the map that you want to place a token. The token size will default to the grid size if it is present. Right click on a token to edit it. From the token menu you can name the token, change its size and color, hide/show the token to players, and link a note.
+` Size  `  controls how big or small the token should be.<br>
+
+` color `  allows you to select the color of your token.<br>
+
+` visible to players `  hides/show the token to players.<br>
+
+` moveable by players ` allows the players to move the token from one location to another.<br>
+
+` lock `  locks a token in place making it immovable. **Locking a token makes it impossible for both the Dungeon master and the players to move or delete the token.**
+
+` delete `  removes a token from the map.
 
 ![token-menu](https://user-images.githubusercontent.com/9096667/87476985-f3b68400-c5ec-11ea-8165-21d0f2c150d1.png)
```

Comment:
I think this is supposed to be the image from here: https://github.com/dungeon-revealer/dungeon-revealer/issues/768#issuecomment-702433295